src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2205,6 +2233,13 @@ void AutomationInstrumentClipView::modEncoderAction(int32_t whichModEncoder, int
 
 						indicator_leds::setKnobIndicatorLevel(whichModEncoder, newKnobPos + kKnobPosOffset);
 
```

Comment:
When holding down a long press or selecting a long press (in pad selection mode) and then subsequently adjusting the values of that long press with the mod encoders, you need to update the mod position to grab the param values at the node last edited so that the changes can be heard while auditioning.